packages/@tinacms/graphql/src/mdx/parse.ts

```diff
@@ -23,16 +23,18 @@ import { TinaField } from '..'
 import type { Content } from 'mdast'
 import { visit } from 'unist-util-visit'
 import type { GraphQLConfig, RichTypeInner } from '../types'
+import type { TinaCloudSchemaEnriched } from '@tinacms/schema-tools'
 import { isNull } from 'lodash'
 import { resolveMediaRelativeToCloud } from '../resolver/media-utils'
 
 export const parseMDX = (
```

Comment:
More of a design comment,

Would it make sense to refactor this into a class, that way `graphQLconfig` and `schema` would not need to be passed around everywhere.